dv/uvm/ibex_dv.f

```diff
@@ -7,6 +7,7 @@
 +define+BOOT_ADDR=2147483648 // 32'h8000_0000
 +define+TRACE_EXECUTION
 +define+RVFI
++incdir+../../shared/rtl
```

Comment:
How about using the same PROJ_DIR variable here as in the next line?

Reply:
ah yes, that's a good point, not sure how I missed using this the first time around. It's been modified to better match the rest of the .f file.